devex/vsextension/ItemTemplates/oehost/host.vstemplate

```diff
@@ -17,7 +17,7 @@
   </WizardExtension>
   <WizardData>
     <packages repository="extension" repositoryId="OpenEnclaveVisualStudioExtension-1">
-      <package id="open-enclave-cross" version="0.15.0" />
+      <package id="open-enclave-cross" version="0.16.1.1" />
```

Comment:
I preemptively released the open-enclave-cross 0.16.1 package on Friday; given an update to the SDK release, a new open-enclave-cross release reflected this update for v0.16.1.1.

Reply:
What does v0.16.1.1 mean?  https://github.com/openenclave/openenclave/blob/master/docs/Community/Releasing.md#version-bump says:

> Version Bump
> A note on naming: please prepend the v for consistency. Tags are v0.7.0 and branches are v0.7.x. We follow [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
> 
> The initial announcement of the upcoming release will be a PR by the release manager to the master branch with the commit to bump the VERSION file to next pre-release, e.g. v0.7.x, and the commit to update the CHANGELOG "Unreleased" section to v0.7.0 (and change the previous to v0.6.0).

SemanticVersioning has no notion of a 4th integer, so I am not sure if this PR if compliant with OE process.

Reply:
v0.16.1.1 refers to the open-enclave-cross package release on NuGet. I wasn't too sure about versioning rules since the initial releases didn't follow one: https://www.nuget.org/packages/open-enclave-cross/0.8.1-c3b6262c

Reply:
Yes it did.   https://semver.org/spec/v2.0.0.html explains that '-' (not '.') is used as a separator for pre-release labels and "+" can also be used as a separator for metadata.   Anything that adheres to the BNF at that link is ok.

Reply:
Ah yes, you're correct. Would you prefer I rebuild and publish with a semantically correct version of the open-enclave-cross package?

Reply:
That would be preferred, yes

Reply:
Updated